tests/core/engine_adapter/docker-compose.yaml

```diff
@@ -60,6 +60,15 @@ services:
     volumes:
       - ./trino/initdb.sql:/docker-entrypoint-initdb.d/initdb.sql
 
+  trino_iceberg_delta_metastore_db:
```

Comment:
Can you leave a comment explaining why it has a separate db? I believe it is because of errors about too many connections so it would be good to document in case someone in the future wants to reduce this down.